build.gradle

```diff
@@ -119,6 +119,7 @@ dependencies {
     compile group: 'org.codehaus.groovy.modules.http-builder', name: 'http-builder', version: '0.7.1'
     compile group: 'io.github.http-builder-ng', name: 'http-builder-ng-okhttp', version: '1.0.3'
     compile group: 'org.apache.httpcomponents', name: 'httpmime', version: '4.5.6'
+    compile 'org.grails.plugins:gorm-logical-delete:2.0.0.M2'
```

Comment:
Once this is merged, there will need to be a lot of updates to the various sql-based queries in `QueryService` and possibly elsewhere. The plugin auto-magically adds a where clause on the deleted field to GORM queries. Raw sql does not do that and we don't do a lot of GORM querying (have been migrating away from that over time).

Reply:
Greg if you want I can do the  sql-based queries  updates in this branch

Reply:
Not necessary for this branch. Updating the sql queries will only affect issues going through the old flow, not the new, so it makes sense to me that this be a separate effort.

Reply:
ok. Thanks!